lib/fastlane/actions/appetize.rb

```diff
@@ -0,0 +1,120 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      APPETIZE_PRIVATE_KEY = :APPETIZE_PRIVATE_KEY
+      APPETIZE_PUBLIC_KEY = :APPETIZE_PUBLIC_KEY
+      APPETIZE_APP_URL = :APPETIZE_APP_URL
+      APPETIZE_MANAGE_URL = :APPETIZE_MANAGE_URL
+    end
+
+    class AppetizeAction < Action
+      APPETIZE_URL_BASE = 'https://api.appetize.io/v1/app/update'
+
+      def self.is_supported?(platform)
+        platform == :ios
+      end
+
+      def self.run(options)
+        require 'net/http'
+        require 'uri'
+        require 'json'
+
+        uri = URI.parse(APPETIZE_URL_BASE)
+        https = Net::HTTP.new(uri.host, uri.port)
+        https.use_ssl = true
+
+        req = Net::HTTP::Post.new(uri.request_uri, initheader={'Content-Type' => 'application/json'})
```

Comment:
Useless assignment to variable - `initheader`.<br>Surrounding space missing for operator `=`.